extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,849 @@
+
+## Octree Example
+
+`tileset.json`:
+
+```json
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "geometricError": 100,
+  "extensionsUsed": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensionsRequired": [
+    "3DTILES_implicit_tiling",
+  ],
+  "extensions": {
+    "3DTILES_implicit_tiling": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "boundingVolume": {
+        "box": [
+          0, 0, 0,
+          100, 0, 0,
+          0, 100, 0
+          0, 0, 100
+        ]
+      },
+      "refine": "ADD",
+      "tilingScheme": "OCTREE",
+      "geometricError": 5000,
+      "subtreeLevels": 3,
+      "maximumLevel": 8,
+      "subtrees": {
+        "uri": "subtrees/${level}/${z}/${y}/${x}/subtree.json"
+      },
+      "content": {
+        "mimeType": "application/octet-stream",
+        "uri": "models/${level}/${z}/${y}/${x}/model.pnts"
+      }
+    }
+  }
+}
+```
+
+`subtrees/0/0/0/0/subtree.json`
+
+```json
+{
+	"buffers": [
+    {
+      "uri": "availability.bin",
+      "byteLength": 74
+    }
+  ],
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteOffset": 0,
+      "byteLength": 10
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 10,
+      "byteLength": 64
+    }
+  ],
+  "tileAvailability": {
+    "bufferView": 0
+  },
+  "contentAvailability": {
+    "constant": 0
+  },
+  "childSubtreeAvailability": {
+    "bufferView": 1
+  }
+}
+```
+
+`subtrees/6/0/0/0/subtree.json`
+
+```json
+{
+	"buffers": [
+    {
+      "uri": "availability.bin",
+      "byteLength": 84
+    }
+  ],
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteOffset": 0,
+      "byteLength": 10
+    },
+    {
+      "buffer": 0,
+      "byteOffset": 10,
+      "byteLength": 10
+    }
+    {
+      "buffer": 0,
+      "byteOffset": 10,
+      "byteLength": 64
+    }
+  ],
+  "tileAvailability": {
+    "bufferView": 0
+  },
+  "contentAvailability": {
+    "bufferView": 1
+  },
+  "childSubtreeAvailability": {
+    "bufferView": 2
+  }
+}
+```
+
+File structure:
+
+```
+|__ tileset.json
+|__ subtrees/
+  |__ 0/
+    |__ 0/
+      |__ 0/
+        |__ 0/
+            |__ subtree.json
+            |__ availability.bin
+  |__ 3/
+    |__ 0/
+      |__ 0/
+        |__ 0/
+          |__ subtree.json
+          |__ availability.bin
+  |__ 6/
+    |__ 0/
+      |__ 0/
+        |__ 0/
+        |__ subtree.json
+        |__ availability.bin
+|__ models/
+  |__ 8/
+    |__ 0/
+      |__ 0/
+        |__ 0/
+          |__ model.pnts
+        |__ 1/
+          |__ model.pnts
+      |__ 1/
+        |__ 0/
+          |__ model.pnts
+        |__ 1/
+          |__ model.pnts
+    |__ 1/
+      |__ 0/
+        |__ 0/
+          |__ model.pnts
+        |__ 1/
+          |__ model.pnts
+      |__ 1/
+        |__ 0/
+          |__ model.pnts
+        |__ 1/
+          |__ model.pnts
+```
```

Comment:
While this is valid, a more common template uri would be `"models/${level}/${z}/${y}/${x}.pnts"`. It's a little weird to have so many `model.pnts` files in practice. It also makes this diagram bigger than it needs to be.